### PR TITLE
Add search by topic page

### PR DESCRIPTION
**Pull request by nvisal1**
***Purpose***
This PR adds a search by topic page to the client.
The page is accessible through the splash on the home page and displays all topics that
are available in the topic service. Clicking any of the topic cards will navigate the user to the browse page
with a filtered display of Learning Objects.

***Note***
This PR cannot be deployed until the topic service and learning object service PRs are deployed.

***Screenshots***
Screen Shot 2020-03-13 at 1 45 04 PM
Screen Shot 2020-03-13 at 1 45 11 PM
Screen Shot 2020-03-13 at 1 45 18 PM

### CODE REVIEW

File: src/app/cube/browse/browse.component.ts [hunk 1]
```diff
@@ -454,6 +456,7 @@ export class BrowseComponent implements AfterViewInit, OnDestroy {
   }
 
   async fetchLearningObjects(query: Query) {
+    console.log(query);
```
Comment (EVogel1999) [inline src/app/cube/browse/browse.component.ts]: Console.log

File: src/app/cube/topic-browse/topic-browse.component.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+.top-bar {
+    background-color: $light-blue;
+    color: white;
+    box-sizing: border-box;
+    padding: 20px;
+    position: relative;
+}
+
+.top-bar__inner {
```
Comment (EVogel1999) [inline src/app/cube/topic-browse/topic-browse.component.scss]: Empty css class

**Comment by pzalep1** [comment]
This is going to get moved to a draft PR. 6/15/2020

**Comment by nvisal1** [comment]
🙁

**Pull request closed without merging**